Italian/scapare/cercare-uscita.md

```diff
@@ -1 +0,0 @@
-Ti alzi per cercare una via d'uscita.
```

Comment:
Thanks,
I renamed the directory to correct the typo 'scapare' into 'scappare', but the result seems to be as I removed a line in cercare-uscita.md, but it is not true